src/style.css

```
@@ -0,0 +1,40 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-position: center;
+  background-image: url("./images/star-active.svg");
+  background-repeat: no-repeat;
+}
+
+.stars--0 > :nth-child(n) {
+  background-image: url("./images/star.svg");
+}
+
+.stars--1 > :nth-child(n+2) {
+  background-image: url("./images/star.svg");
+}
+
+.stars--2 > :nth-child(n+3) {
+  background-image: url("./images/star.svg");
+}
+
+.stars--3 > :nth-child(n+4) {
+  background-image: url("./images/star.svg");
+}
+
+.stars--4 > :nth-child(n+5) {
+  background-image: url("./images/star.svg");
+}
+
+.stars--5 > :nth-child(n+6) {
+  background-image: url("./images/star.svg");
+}
```

```suggestion
.stars--0 > :nth-child(n),

.stars--1 > :nth-child(n+2),
.stars--2 > :nth-child(n+3),
.stars--3 > :nth-child(n+4),
.stars--4 > :nth-child(n+5),
.stars--5 > :nth-child(n+6) {
  background-image: url("./images/star.svg");
}
```
don't repeat yourself )